configs/in-development/nessie-0.6-iceberg-flink-0.12.yml

```diff
@@ -18,11 +18,10 @@
 
```

Comment:
can you remove the `@pytest.mark.skip` from `test_nessie_iceberg_flink_demo_nba.py` so that those tests are executed with that Iceberg version?

Reply:
flink is in development and waiting for `0.12.0`. I am just replacing the hash.

Reply:
ah sorry, I thought there's a new Iceberg version